packages/date-picker/src/DatePicker/DatePicker.scss

```diff
@@ -3,119 +3,130 @@
 @import "~@kaizen/design-tokens/sass/spacing";
 @import "~@kaizen/design-tokens/sass/border";
 @import "~@kaizen/design-tokens/sass/shadow";
-@import "~@kaizen/draft-button/KaizenDraft/Button/styles";
 
 // Vars
-$border-radius: 3px;
+$border-radius: 7px;
+$button-height: 48px;
+$button-base-padding-horizontal: $spacing-sm;
+$button-icon-size: 1.25rem; // 20px
+
+$disabled-opacity: 0.5;
```

Comment:
Usually we use 0.3 for disabled opacity. I haven't had a good look but just want to make sure this is right?

Reply:
Yeah this is correct, the `InputSearch` has the same. It's a bit weird but the calendar icon is less opaque than the rest so you need to split it up.